pkg/descheduler/node/node.go

```diff
@@ -18,13 +18,18 @@ package node
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/validation"
+	_ "k8s.io/apiserver/pkg/features"
```

Comment:
'_' as package alias?

Reply:
Line 28 should have been removed, it's gone now.